tests/test_ndjson.py

```diff
@@ -467,6 +467,31 @@ def test_ndjson_import_with_shortnames_in_identifier(here, engine, hr_schema, db
     assert records == [{"sbi_act_nr": "12"}, {"sbi_act_nr": "16"}]
 
 
+def test_ndjson_import_with_attributes_on_relation(
+    here, engine, brk_schema, verblijfsobjecten_schema, dbsession
+):
+    """Prove that extra attributes on a relation does not interfer with ndjson import."""
```

Comment:
"""Prove that extra attributes on a relation do not interfere with ndjson import."""